certbot-apache/certbot_apache/tests/http_01_test.py

```diff
@@ -134,6 +135,17 @@ def test_perform_3_achall_apache_2_2(self):
     def test_perform_3_achall_apache_2_4(self):
         self.combinations_perform_test(num_achalls=3, minor_version=4)
 
+    def test_activate_disabled_vhost(self):
+        vhosts = [v for v in self.config.vhosts if v.name == "certbot.demo"]
+        achalls = [
+            achallenges.KeyAuthorizationAnnotatedChallenge(
+                challb=acme_util.chall_to_challb(
+                    challenges.HTTP01(token=((b'a' * 16))),
+                    "pending"),
+                domain="certbot.demo", account_key=self.account_key)]
+        vhosts[0].enabled = False
```

Comment:
I'm not sure what's going on here, but if I set this value to `True`, tests still pass. This seems bad. Do you know why this is happening?

Reply:
The actual test assert is now moved to this test function.